compiler/control/OMROptions.cpp

```diff
@@ -183,6 +183,7 @@ TR::OptionTable OMR::Options::_jitOptions[] = {
                                TR::Options::setStaticNumeric, (intptrj_t)&OMR::Options::_compThreadCPUEntitlement, 0, "F%d", NOT_IN_SUBSET },
    {"concurrentLPQ", "M\tCompilations from low priority queue can go in parallel with compilations from main queue", SET_OPTION_BIT(TR_ConcurrentLPQ), "F", NOT_IN_SUBSET },
    {"conservativeCompilation","O\tmore conservative decisions regarding compilations", SET_OPTION_BIT(TR_ConservativeCompilation), "F"},
+   {"continueAfterILValidationError", "O\tAll other things held constant, do not abort upon encountering an ILValidation failure.", SET_OPTION_BIT(TR_ContinueAfterILValidationError), "F"},
```

Comment:
Done.

> Is it needed? 

No, not yet at least. So good catch!

But I do believe once we introduce the notion of `Strict` vs `Lenient` Rules, we will need to have something like this. Since if a `Rule` is set to `Lenient`, we do indeed _continue after an IL Validation Error_ even if the "continueAfterILValidationError" option is not set. I will have to give it some more thought, once that functionality is added. 

The original intent was for this option to be able to able to override this behaviour if required. As in, even if a Rule was set as a `Strict` one (`Rules` for which we abort compilation rather than just emit an diagnostic message), this option would allow us to override that behaviour and continue compilation. However I have decided to add the `Strict` | `Lenient` split as a followup `PR`.
So although compilation is expected to abort under conditions, can fail or the relation can be abolished by intervening factors.

The "All other things held constant" part was essentially alluding to that. Thanks for catching that. I will find a better way to phrase it next time.

Reply:
Done.